org.eclipse.winery.topologymodeler.ui/src/app/models/nodeVisualsModel.ts

```diff
@@ -0,0 +1,18 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache Software License 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+
+export interface NodeVisualsModel {
+    color: any;
```

Comment:
I think this is a string, isn't it?
The same holds for the `imageUrl`

Reply:
done